onnx/defs/nn/defs.cc

```diff
@@ -851,7 +865,7 @@ computes the output.)DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     Conv,
-    11,
+    14,
```

Comment:
For this opset bump, do you only modify the document? Or you will make other updates? Thanks.

Reply:
(1) I modify the document (2) I added two shape inference test for Conv-14 and ConvTranspose-14. That is all.
For these two shape inference test, it already works well without any change on shape inference.
This means that the shape inference already has this Conv-14 and ConvTranspose-14 behavior.
Previously we don't have shape inference test for this scenario, so there was no validation previously.